frontend/src/components/Messenger/Messenger.cy.js

```diff
@@ -0,0 +1,39 @@
+import { mount } from 'cypress/react18';
+import Messenger from '.';
+
+describe('Messenger', () => {
+  beforeEach(() => {
+    mount(<Messenger />);
+  });
+
+  const openCloseButton = '[class^="Messenger_openCloseButton"]';
+
+  it('It opens and closes the messenger', () => {
+    cy.get(openCloseButton).click();
```

Comment:
```suggestion
    cy.get(openCloseButton).click();

```

Separe ação e resultado esperado com uma linha em branco, pare que fique claro o que é _Act_ e o que á _Assert_.

Mais sobre o padrão 3A no último encontro.